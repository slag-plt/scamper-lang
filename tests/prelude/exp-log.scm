(exp 13)
(exp 0.71)
(exp 111)
(exp 4)
(exp 6.1)
(exp -5)
(exp -0.69)
(exp 89)
(exp 0)
(exp -0.10000000000000009)
(log 13)
(log 0.71)
(log 111)
(log 4)
(log 6.1)