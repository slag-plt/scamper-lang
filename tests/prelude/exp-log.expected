442413.3920089205
2.0339912586467506
1.609487066961518e+48
54.598150033144236
445.85777008251677
0.006737946999085467
0.5015760690660556
4.489612819174345e+38
1
0.9048374180359595
2.5649493574615367
-0.342490308946776
4.709530201312334
1.3862943611198906
1.8082887711792655
